packages/components/tests/dummy/app/templates/components/button.hbs

```diff
@@ -344,7 +344,13 @@
           <div>
             <span class="dummy-text-small">{{capitalize size}}/{{state}}:</span>
             <br />
-            <Hds::Button @icon="plus" @text={{capitalize state}} @size={{size}} @color={{color}} class="is-{{state}}" />
+            <Hds::Button
+              @icon="plus"
+              @text={{capitalize state}}
+              @size={{size}}
+              @color={{color}}
+              mock-state-value={{state}}
```

Comment:
here you can see how one of these mock states is declared on a component, but **only** in the dummy documentation page